sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationprovider.py

```diff
@@ -62,7 +70,7 @@
 
 
 @overload
-def load(
+def load(  # pylint: disable=docstring-keyword-should-match-keyword-only
```

Comment:
Why we disable it rather than fixing the doc?

Reply:
This was a suggestion from Anna a while back. We use to have an object that took in a set of Key Vault options, but we "removed" it in place of adding kwarg options. To make it not a breaking change we removed that doc for it, but still support the option.